### PR TITLE
Migrate away from using poise-python

### PR DESCRIPTION
**Pull request by andrewda**

### CODE REVIEW

**Comment by andrewda** [comment]
[Migrate away from poise-python](https://trello.com/c/22vtHTq4/1528-migrate-away-from-poise-python)

File: recipes/default.rb [hunk 1]
```diff
@@ -17,6 +17,7 @@
 # limitations under the License.
 include_recipe 'yum-epel'
 include_recipe 'osl-mysql::client'
+include_recipe 'base::python'
 
 node.override['user']['home_dir_mode'] = '2750'
 node.default['poise-python']['provider'] = 'system'
```
Comment (ramereth) [inline recipes/default.rb]: Please remove this reference.

**Comment by andrewda** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v2.3.1.  Have a nice day!

**Pull request merged**